scripts/datadog_wrapper

```diff
@@ -32,11 +32,15 @@ then
   then
     echo "Configuring for the .NET runtime!"
   fi
-  # Handle the CORECLR_PROFILER_PATH
-  DEFAULT_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
-  ARM64_PATH=/opt/datadog/linux-arm64/Datadog.Trace.ClrProfiler.Native.so
-  X64_PATH=/opt/datadog/linux-x64/Datadog.Trace.ClrProfiler.Native.so
-  PROFILER_PATHS=("${DEFAULT_PATH}" "${ARM64_PATH}" "${X64_PATH}")
+  # Handle the CORECLR_PROFILER_PATH:
+  # Load the tracer library directly (Datadog.Tracer.Native.so),
+  # which avoids loading the shared loader (Datadog.Trace.ClrProfiler.Native.so)
+  # since we don't support Continuous Profiling yet.
+  # Keep trying Datadog.Trace.ClrProfiler.Native.so for backwards compatibility.
+  PROFILER_PATHS=("/opt/datadog/linux-x64/Datadog.Tracer.Native.so"
```

Comment:
I have two small concerns.
- If we start to add extra valid path, old version of the extension may not work with the new version of the layer (if the new layers no longer copy the `Datadog.Tracer.Native.so` as the `Datadog.Trace.ClrProfiler.Native.so`
- I don't know how hard it would be to support the continuous profiling feature in lambda. Maybe we can support it in the near future?

Reply:
Good catch! I considered backwards compatibility in one direction (newer extension with older dotnet layer), but not the opposite direction.

Reply:
@joeyzhao2018, I restored `/opt/datadog/Datadog.Trace.ClrProfiler.Native.so`, re-ordered the paths, and added comments. This should now cover all the cases and be backwards and forwards compatible.